app/controllers/admin/config/branding_controller.rb

```diff
@@ -38,4 +38,36 @@ def logo
       end
     end
   end
+
+  def fonts
```

Comment:
I do think it is still good to have controller tests that covers the possible responses:

1. Non-admin hitting this route
2. Responses on failed policy